gui/wxpython/gui_core/wrap.py

```diff
@@ -122,6 +124,22 @@ def __init__(self, *args, **kwargs):
         wx.SpinCtrl.__init__(self, *args, **kwargs)
 
 
+class FloatSpin(fs.FloatSpin):
```

Comment:
Looks good, but please add also the SetToolTip function as in other classes to the FloatSpin class to keep it consistent. The wrappers were originally developed for compatibility between wx3 and 4 and although that may not be such a big concern now, it's generally good idea to have wrappers around GUI library for various OS/versions compatibility issues.

Reply:
Okay I added some wrapper classes.